src/views/projects/Browser.svelte

```diff
@@ -41,13 +46,17 @@
   // Whether the mobile file tree is visible.
   let mobileFileTree = false;
 
+  const api = new HttpdClient(seedBaseUrl);
+
   const loadBlob = async (path: string) => {
     if (state.status === Status.Loaded && state.path === path) {
       return state.blob;
     }
 
     const promise =
-      path === "/" ? project.getReadme(commit) : project.getBlob(commit, path);
+      path === "/"
+        ? api.project.getReadme(project.id, commit)
+        : api.project.getBlob(project.id, commit, path);
 
     state = { status: Status.Loading, path };
     state = { status: Status.Loaded, path, blob: await promise };
```

Comment:
Unrelated to this PR but these lines were really hard for me to understand. I think they obscure what is going on here. May I suggest the following:
```typescript
state = { status: Status.Loading, path };

let blob 
if  (path === "/") {
  blob = await api.project.getReadme(project.id, commit)
} else {
  blob = await api.project.getBlob(project.id, commit, path);
}

state = { status: Status.Loaded, path, blob };
return blob;
```

Reply:
471cf129735ba24a85f776c521cced9faf477686